inventory.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+title: "Inventorying Digital Media Items"
+---
+
+## Overview
+
+Digital media items should be inventoried in the Digital Media Log as they are encountered during processing. In order to save a digital media item record, you must supply the RefId of the appropriate item-level component and retrieve the item and resource titles. For minimal processing, you may supply the RefId of the series-level or subseries-level component. Digital media items should be described as a sibling relationship to their parent units and should have the same box and/or folder instances. Do not create a child relationship to represent the association between the digital media item and the parent unit.
+
+## Setup
+
+The Digital Media Log is located at
+[dm-log.rockarch.org](http://dm-log.rockarch.org/).
+
+## Instructions
+
+1.  Add a new digital media item record.
+2.  The digital media item ID field will autopopulate.
+3.  Copy the RefId for the parent component from ArchivesSpace into the
+    "ArchivesSpace RefID" field and click "Find in ArchivesSpace." (Make
+    sure the component has been saved in ArchivesSpace.)
+4.  The component and resource fields should autopopulate. If they do
+    not, contact the administrator.
+5.  Choose the appropriate format from the drop down menu.
+6.  If you are ready to transfer the item, continue to [transfer setup](transfer-instructions#transfer-overview-&-setup)
+
+## Recording Transfers in Digital Media Log
```

Comment:
I'd suggest moving this to the "Transfer Instructions" page, since that's where users will actually be doing this.